RevenueCatUI/Data/PaywallViewConfiguration.swift

```diff
@@ -22,7 +22,7 @@ struct PaywallViewConfiguration {
     let useDraftPaywall: Bool
     var introEligibility: TrialOrIntroEligibilityChecker?
     var purchaseHandler: PurchaseHandler
-    var locale: Locale
```

Comment:
I realized that the `locale` of `PaywallViewConfiguration` was actually redundant. It was always holding `Locale.current`, as the `init` had `.current` as the default value for the `locale` parameter and this `locale` parameter was never used.
That's why I've replaced it with `preferredLocale`, which will now potentially come from the intregrator's side

Reply:
In those places where a `locale` is non-optional (basically in v1 paywalls, if I'm not mistaken), I've replaced the use of `self.locale` to `self.preferredLocale ?? .current` to keep the existing behavior if `preferredLocale` is not specified